docs/building-a-form/quick-start-example-formjs-file.md

```diff
@@ -28,83 +28,90 @@ Use this example `form.js` file to build a basic form. For more information abou
-          // from being sent to the server, such as a question that's only used to reveal other questions, or to group related
-          // questions together to be conditionally revealed that aren't in an object in the schema.
-          'view:field2': {
-            type: 'string'
+
+      // The object that contains the pages in each chapter.
+      pages: {
+        // Each property is the key for a page, and should be unique across chapters.
+        pageOne: {
+            // The URL for the page.
+          path: 'some-path',
+
+          // The title of the page that renders on the review page.
+          title: '',
+          // `title` can also be a function that receives the current data as a parameter:
+          title: formData => `A title for ${formData.thing}`,
+
+          // Any initial data that should be set for the form.
+          initialData: {
+            field1: 'Default string'
           },
-          'view:artificialGroup'{
+
+          // Specifies that a page will turn its schema into a page for each item in an array, such as an array of children with a page
+          // for each child. The schema/uiSchema for this type of page should be built as usual for an array field.
+          showPagePerItem: true,
+          // The path to the array.
+          arrayPath: 'children',
+          // Items in the array that should not have a page.
+          itemFilter: () => true,
+          // You must specify a path with an :index parameter.
+          path: 'some-path/:index',
+
+          // The JSON schema object for the page, following the JSON Schema format.
+          schema: {
             type: 'object',
+            // A schema's properties refer to definitions. For example:
+            //   "homePhone": { "$ref": "#/definitions/phone" }
+            // In the configuration file, the definition for `phone` must be added into definitions in order to be parsed correctly and added to `homePhone`.
+            definitions: {},
             properties: {
-              // `view:artificialGroup` is flattened. `subField1` and `subField2` are siblings of `field1` when sent to the API.
-              subField1: {
+              field1: {
                 type: 'string'
               },
-              subField2: {
-                type: 'boolean'
+              // Fields of type `string`, `boolean`, `number`, and `array` that begin with `view:` are excluded from data that's sent to
+              // the server. Instead, their children are merged into the parent object and sent to the server. Use these to exclude fields
+              // from being sent to the server, such as a question that's only used to reveal other questions, or to group related
+              // questions together to be conditionally revealed that aren't in an object in the schema.
+              'view:field2': {
+                type: 'string'
+              },
+              'view:artificialGroup'{
```

Comment:
There is a small typo here that I know existed before, but just catching it now: we're missing the `: ` after the key name and before the open curly bracket.